fala/apps/adviser/tests/test_fala_views_and_functions.py

```diff
@@ -214,6 +214,26 @@ def test_search_parameters_box_contains_only_organisation_and_categories(self):
         self.assertEqual(content, "Organisation: fooLegal problem: Debt, Education   Change search")
 
 
+class SingleCategoryResultsPageTest(SimpleTestCase):
```

Comment:
Question - should this not be in `fala/apps/adviser/tests/test_single_category_search_view.py` ?

Reply:
The reason I have put them in the results page tests is because I am testing the content on the results page, whereas the 'Search' page is the landing page, and those tests are checking the content that displays there, the redirecting etc

Reply:
And the code I have added is directly in the results template, so seemed reasonable

Reply:
That make sense. Should we call the test/class `ResultsPageWithSingleCategorySearch` just so it's in keeping with the other naming of the tests? Ignore this if you think it's fine as is.

Reply:
the reason I didn't name it 'single category search' and just put 'single category' is because I feel that the naming convention could change seeing as we are going to support multiple categories as the feature. But these tests are for a specific single category. Does that make sense? I didn't really want to refer to 'single category search' as a thing. That was my logic. 

And then ResultsPageWithSingleCategory didn't feel quite right because the results do take a single category at the minute, where as this is something a little different. But I'm really splitting hairs. It is sort of a SingleCategoryResultsPage in my head.